attestation-agent/attestation-agent/Cargo.toml

```diff
@@ -76,6 +76,7 @@ all-attesters = [
     "cca-attester",
 ]
 tdx-attester = ["kbs_protocol?/tdx-attester", "attester/tdx-attester"]
+tdx-attester-oot = ["kbs_protocol?/tdx-attester-oot", "attester/tdx-attester", "attester/tdx-attest-dcap-ioctls"]
```

Comment:
Can we add a comment about what this is. Is `oot` the best name for the feature? This is referring to a non-upstream kernel? Maybe we should just call it `-libtdx` or something.

Reply:
I renamed it to `tdx-attester-libtdx` and added a README entry.